docs/getting-started/release-notes/v2_13_0.md

```diff
@@ -0,0 +1,110 @@
+# Version 2.13.0 (December 2023)
+
+Welcome to the Zowe Version 2.13.0 release!
+
+See [New features and enhancements](#new-features-and-enhancements) for a full list of changes to the functionality. See [Bug fixes](#bug-fixes) for a list of issues addressed in this release.
+
+**Download v2.13.0 build**: Want to try new features as soon as possible? You can download the v2.13.0 build from [Zowe.org](https://www.zowe.org/download.html).
+
+## New features and enhancements
+
+Zowe Version 2.13.0 contains the enhancements that are described in the following topics.
+
+### Zowe installation and packaging
+
+### Zowe Application Framework
+
+### Zowe API Mediation Layer
+
+### Zowe CLI
+
+#### Zowe CLI (Core)
+
+- Incorporated all source code from the `zowe/imperative` Github repository into the `zowe/zowe-cli` repository for a more streamline code architecture. This change should have no user impact.
+- Deprecated `getDataSet` in the `zosfiles` command group utility functions. Use `zosfiles` SDK's `ZosFilesUtils.getDataSetFromName` command instead. ([#1696](https://github.com/zowe/zowe-cli/issues/1696))
```

Comment:
@adam-wolfe - wanted to check whether this deprecation should be listed under "new features." Also, can we explain why we did this?

Reply:
I think the motivation behind this is to reduce code duplication by removing a function that is defined in the zosfiles command group utility functions that also exists within the zosfiles SDK. In V2, both functions will be available, but in V3, only the function from the zosfiles SDK will be available.

Reply:
You could change it to something like: "Deprecated `getDataSet` in the `zosfiles` command group utility functions. `getDataSet` will be removed in V3. Use the `zosfiles` SDK's `ZosFilesUtils.getDataSetFromName` function instead."